<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenu</class>
 <widget class="QWidget" name="MainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="lay">
   <item>
    <widget class="QPushButton" name="tutorial_button">
     <property name="text">
      <string>Обучение</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="training_button">
     <property name="text">
      <string>Тренировка</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
